src/numberArray.js

```diff
@@ -0,0 +1,15 @@
+export default function(a, b) {
+  if (!b) b = [];
+  var na = a ? Math.min(b.length, a.length) : 0,
+    c = b.slice();
+  let i;
```

Comment:
Oops, this little `let` here would break everything, unless you intend for this to only ship with d3-interpolate 2.0.

Reply:
I'm so bad at this, I wish we had automated tests for all the modules.